dart/lib/flat_buffers.dart

```diff
@@ -299,9 +299,7 @@ class Builder {
 
   /// End the current table and return its offset.
   int endTable() {
-    if (_currentVTable == null) {
-      throw new StateError('Start a table before ending it.');
-    }
+    _ensureCurrentVTable();
```

Comment:
nit: consider making these methods return a bool and then just wrapp them in the `assert`, so the whole thing just gets shaken out when asserts are disabled more easily.

Reply:
done, albeit with a single getter this time